tests/unit/storage/utils.js

```diff
@@ -0,0 +1,48 @@
+const assert = require('assert');
+const { serializeCircJSON } = require('../../../lib/storage/utils');
+
+const jsonObj = {
```

Comment:
No big deal, but the naming can be confusing as this is a javascript object, not a JSON object.